build.sbt

```diff
@@ -4,7 +4,7 @@ import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
 val appName = "compliance-documents-api"
 
 majorVersion := 0
-scalaVersion := "2.13.12"
+scalaVersion := "3.3.4"
```

Comment:
Can you correct the Scoverage settings in this PR please? https://build.tax.service.gov.uk/job/DDCW%20Live%20Services/job/IDRS/job/compliance-documents-api-pr-builder/scoverage/
They're not working correctly as they're including files/packages they shouldn't. Also coverage is below 80% and it's not failing - though I think excluding the correct files/packages should fix this